src/style.css

```diff
@@ -0,0 +1,122 @@
+html {
+  --content-width: 166px;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.product-card {
```

Comment:
add padding for all sides to the card, instead of adding horizontal margins to all elements